src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -268,15 +270,17 @@ private static Team teamOf(uk.gov.justice.digital.delius.jpa.standard.entity.Off
          * This only populates a subset of team data - this is currently indexed in elasticsearch so if this is modified
          * then it will lead to inconsistent data and the index will need to be rebuilt.
          */
-        return Optional.ofNullable(offenderManager.getTrustProviderTeam())
+        return Optional.ofNullable(offenderManager.getTeam())
                 .map(tpt -> Team.builder()
+                        .code(tpt.getCode())
                         .description(tpt.getDescription())
                         .telephone(tpt.getTelephone())
                         .district(Optional.ofNullable(tpt.getDistrict()).map(
                                 d -> KeyValue.builder()
                                         .code(d.getCode())
                                         .description(d.getDescription()).build())
                                 .orElse(null))
+                        .localDeliveryUnit(keyValueOf(tpt.getDistrict()))
```

Comment:
essentially a backward compatible way of renaming district LDU, again to be consistent  with offender manages endpoint